### DISCUSSION TITLE
Drone-mounted GPR/IPR resources

### DISCUSSION THREAD

**Discussion by hsinc** [dannyjacobs/ECHO#62] [General]
Here's a collection of resources describing drone-mounted radar units:
- [GPR made with a Red Pitaya](https://web.archive.org/web/20211105044946/http://au.feri.um.si/losdv/index.php/en/nato-sps-about) (original link no longer exists, available only on the Wayback Machine)
- [Narod and Clarke, Miniature high-power impulse transmitter for radio-echo sounding, 1994](https://www.cambridge.org/core/services/aop-cambridge-core/content/view/EE265E7F4D7F54FB0A9A06FD78206AFD/S002214300000397Xa.pdf/div-class-title-miniature-high-power-impulse-transmitter-for-radio-echo-sounding-div.pdf)
- [Mingo and Flowers, An integrated lightweight ice-penetrating radar system, 2017](https://www.cambridge.org/core/journals/journal-of-glaciology/article/an-integrated-lightweight-icepenetrating-radar-system/C4CAFD9F66EC5FE990AE873FBEAF4CF5)
- [Mingo et al., A stationary impulse-radar system for autonomous deployment in cold and temperate environments, 2020](https://www.cambridge.org/core/journals/annals-of-glaciology/article/stationary-impulseradar-system-for-autonomous-deployment-in-cold-and-temperate-environments/7AA6AF76E87AA2E8AFE4317152BC1FB5)

**Comment by mikehetu** [comment]
Found an updated [link](https://au.feri.um.si/losdv/en/gpr_radar/) for the Red Pitaya GPR
